src/pages/studyView/StudyViewPage.tsx

```diff
@@ -221,35 +222,18 @@ export default class StudyViewPage extends React.Component<
     }
 
     async getBookmarkUrl(): Promise<ShareUrls> {
-        let bitlyResponse;
-
-        // now lets shorten with bityly, if we have key
-        // WE ARE DISABLING BITLY PENDING DISCUSSION
         console.log(AppConfig.serverConfig.bitly_access_token);
```

Comment:
thus is probably my console, but lets kill it.  sorry.

Reply:
sounds good